packages/weston.rb

```diff
@@ -60,40 +63,33 @@ class Weston < Package
   depends_on 'xdg_base' => :build
   depends_on 'zlibpkg' # R
 
-  def self.build
-    system "LIBRARY_PATH=#{CREW_LIB_PREFIX} meson setup #{CREW_MESON_OPTIONS} \
-        -Dbackend-default=wayland \
+  meson_options "-Dbackend-default=wayland \
         -Dbackend-drm=true \
         -Dbackend-rdp=false \
         -Dcolor-management-lcms=false \
-        -Dlauncher-libseat=true \
         -Dremoting=true \
         -Dshell-ivi=false \
         -Dsystemd=false \
-        -Dxwayland-path=#{CREW_PREFIX}/bin/Xwayland \
-        builddir"
-    system 'meson configure --no-pager builddir'
-    system "#{CREW_NINJA} -C builddir"
+        -Dxwayland-path=#{CREW_PREFIX}/bin/Xwayland"
+
+  def self.install
+    system "DESTDIR=#{CREW_DEST_DIR} #{CREW_NINJA} -C builddir install"
     File.write 'weston.ini', <<~WESTON_INI_EOF
       [core]
       xwayland=true
 
       [terminal]
       font=Cousine
-      EOF"
     WESTON_INI_EOF
```

Comment:
The weston.ini could go in `def self.build`?

Reply:
Actually, now that I think of it, that would hinder the `meson_options` assignment.